WTWSMS/history/characters/avar.txt

```diff
@@ -54,21 +54,6 @@
 } 
 
 
-33079 = {
-	name="Bayan"
-	# Bayan Bayanid
-	dynasty=10424388  
-	religion="tengri_pagan"
-	culture="avar"
-	father=31048 
-	530.1.1={
-		birth="530.1.1"
-	}
-	602.1.1={
-		death="602.1.1"
-	}
-}
-
```

Comment:
The other Bayan doesn't have any father entry.

Reply:
> The other Bayan doesn't have any father entry.

This deleted Bayan was descended from Slavic rulers, which IMO is very strange

Reply:
For the Avars? Have you checked what genealogy tree that made?

Reply:
> For the Avars? Have you checked what genealogy tree that made?

There was the two Bayans and i preserved the one who owns Avar khaganate and have a son. Second Bayan was son of the Slavic ruler. So, now the Avar dynasty starts with Bayan I.

Reply:
Ok.